<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>830</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Корзина 1</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QListWidget" name="Basket1"/>
         </item>
         <item>
          <widget class="QPushButton" name="moveToSecondBtn">
           <property name="text">
            <string>Переложить шар в корзину 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Корзина 2</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QListWidget" name="Basket2"/>
         </item>
         <item>
          <widget class="QPushButton" name="moveToFirstBtn">
           <property name="text">
            <string>Переложить шар в корзину 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="basketLayout3">
      <item>
       <widget class="QListWidget" name="Common"/>
      </item>
      <item>
       <widget class="QPushButton" name="takeTwoBtn">
        <property name="text">
         <string>Извлечь 2 шара из случайной корзины</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
